sale_variant_configurator/models/sale_order.py

```diff
@@ -63,20 +64,19 @@ def create(self, vals):
 
     @api.onchange("product_tmpl_id")
     def _onchange_product_tmpl_id_configurator(self):
-        res = super()._onchange_product_tmpl_id_configurator()
+        res = super()._onchange_product_tmpl_id_configurator() or {}
```

Comment:
I find it a bit weird to call super or {}

Reply:
I added because sometimes, res returns None, so when it tries to add domain = res.setdefault("domain", {}), gives an error message as res is not inicialized